react-native-picker-module.d.ts

```diff
@@ -5,9 +5,14 @@
 
 import React from "react"
 
+export type PickerItem<V = any> = string[] | {
```

Comment:
Aren't there a limited number of items value can be? I would think only `number` or `string`

Reply:
Yes, you are correct. I don't use this library any more, looks like it's no longer maintained